src/EnergyPlus/CMakeLists.txt

```diff
@@ -301,7 +301,7 @@ SET( SRC
   Humidifiers.cc
   Humidifiers.hh
 )
-
+ 
```

Comment:
I found "Trailing Whitespace Visualizer" for an extension Visual Studio that does this.

https://visualstudiogallery.msdn.microsoft.com/a204e29b-1778-4dae-affd-209bea658a59